itm/dns_apps_test.go

```diff
@@ -51,7 +28,7 @@ func TestErrorIssuingPostOnCreateDNSApps(t *testing.T) {
 	if app != nil {
 		t.Error("Expected nil result")
 	}
-	expectedError := "Post https://itm.cloud.com:443/api/v2/config/applications/dns.json?publish=false: foo"
+	expectedError := "Post \"https://itm.cloud.com:443/api/v2/config/applications/dns.json?publish=false\": foo"
```

Comment:
nit: you can use backtick here for readability

```
`Post "https://itm.cloud.com:443/api/v2/config/applications/dns.json?publish=false"`
```

Reply:
I will change, thanks for the tips